### PR TITLE
add photo widget 

### LINKED ISSUE #3: Profile displays large photo(s)
Each access area shows a prominent photo (or set of photos). Some will be high quality, some not. Areas that are under construction will not have photos, so we'll need to figure out how to handle those.

### PR DESCRIPTION
**Pull request by chriswhong**
This PR
- adds images to the profiles model using the github contents api
- displays images using ember-cli-slick (a wrapper for the popular slick jquery slider)

Closes #3
- [x] Documentation has been created/updated to reflect changes in this PR
- [x] Tests have been written to prevent regressions or breaking changes

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -24,7 +24,12 @@ jobs:
           # fallback to using the latest cache if no exact match is found
           - v1-dependencies-
 
+      - run:
+          name: Install bower
+          command: 'sudo npm install -g bower'
+
       - run: yarn install
+      - run: bower install
```
Comment (allthesignals) [inline .circleci/config.yml]: Is there some reason we should be using `bower`? It is still maintained, but they recommend [migrating away from it](https://bower.io/blog/2017/how-to-migrate-away-from-bower/). Can we install Slick some other way?

File: app/routes/profiles/show.js [hunk 2]
```diff
@@ -1,7 +1,15 @@
...
 
 export default class ShowProjectRoute extends Route {
-  model({ id }) {
-    return this.store.findRecord('profile', id, { reload: true });
+  async model({ id }) {
+    const model = await this.store.findRecord('profile', id, { reload: true });
+    const images = await fetch(`https://api.github.com/repos/nycplanning/labs-waterfront-access-photos/contents/${id}`)
+      .then(d => d.json())
+
+    if (!images.message) {
```
Comment (allthesignals) [inline app/routes/profiles/show.js]: Is this for error handling?

File: app/routes/profiles/show.js [hunk 3]
```diff
@@ -1,7 +1,15 @@
...
+      model.set('images', images.map(d => d.download_url))
```
Comment (allthesignals) [inline app/routes/profiles/show.js]: FYI, this puts the model in "dirty" state which could have some unexpected consequences in the future. You could avoid this by making it a regular classic property (without the `attr()` decorator), or having a computed on the model that returns this async promise.

Comment (andycochran) [reply app/routes/profiles/show.js]: Yup. When switching from a profile w/ an image to one w/o, `model.images` isn't null.

Comment (allthesignals) [reply app/routes/profiles/show.js]: @andycochran hmmm what do you mean? This would only apply to models of the same ID.

**Pull request merged**